Themes/default/index.template.php

```diff
@@ -651,20 +651,20 @@ function template_quickbuttons($list_items, $list_id = null, $output_method = 'e
 
 	// Print the quickbuttons
 	$output = '
-		<ul' . (!empty($list_id) ? ' id="quickbuttons_'.$list_id.'"' : '') . ' class="quickbuttons">';
+		<ul class="quickbuttons' . (!empty($list_class) ? ' ' . $list_class : '') . '">';
```

Comment:
the custom class should probably be in the format of `quickbuttons_{class}` otherwise there is a chance there will be conflicts.

Reply:
Yeah, okay.